src/containers/WindowSideBarAnnotationsPanel.js

```diff
@@ -0,0 +1,41 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  getIdAndContentOfAnnotations,
+  getSelectedCanvas,
+  getSelectedCanvasAnnotations,
+  getAnnotationResourcesByMotivation,
+} from '../state/selectors';
+import { WindowSideBarAnnotationsPanel } from '../components/WindowSideBarAnnotationsPanel';
+
+/**
+ * mapStateToProps - to hook up connect
+ * @memberof WindowSideBarAnnotationsPanel
+ * @private
+ */
+const mapStateToProps = (state, { windowId }) => ({
+  annotations: getIdAndContentOfAnnotations(
+    getAnnotationResourcesByMotivation(
+      getSelectedCanvasAnnotations(state, getSelectedCanvas(state, windowId).id),
+      ['oa:commenting', 'sc:painting'],
```

Comment:
Not sure if these are the proper motivations to begin with.  The design has a drop-down that will allow us to switch between particular motivations (which would then be a prop here), but that's for future work.